apps/root/src/pages/aggregator/swap-container/components/swap-settings/index.tsx

```diff
@@ -46,6 +47,8 @@ const StyledOverlay = styled.div`
   padding: 24px;
   display: flex;
   overflow: auto;
+  background-color: ${({ theme: { palette } }) => colors[palette.mode].background.quarteryNoAlpha};
+  border-radius: inherit;
```

Comment:
this necessary?

Reply:
No it's not, I had sharp corners overflowing the container but they are gone now 🤔